src/pages/plan/utils/WaypointDispatcherResolver.ts

```
@@ -0,0 +1,36 @@
+import { socketEventBus } from '../hooks/useSocketHandler';
+import type {
+  WayPointCreateType,
+  WayPointInitType,
+  WayPointResponseType,
+  WayPointUpdateType,
+} from '../types/WaypointResponseBodyType';
+
+export function WaypointDispatcherResolver(message: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)